### PR TITLE
[OTX] Update OpenVino version to 2022.3.0

### PR DESCRIPTION
**Pull request by sovrasov**
- [ ] Waiting for a new NNCF release with the latest OV support

### CODE REVIEW

File: requirements/openvino.txt [hunk 1]
```diff
@@ -3,5 +3,5 @@
 nncf==2.3.0
 onnx==1.10.1
 openmodelzoo-modelapi==2022.2.0
```
Comment (goodsong81) [inline requirements/openvino.txt]: Please try the next version that I've just uploaded :)
image

**Review by goodsong81 — CHANGES_REQUESTED** [review]

**Comment by goodsong81** [comment]
@sovrasov I heard from Mark that this change should be targeting Geti1.2 so that we need to change the base branch to develop.

**Comment by sovrasov** [comment]
I have a PR to develop as well https://github.com/openvinotoolkit/training_extensions/pull/1393

**Comment by goodsong81** [comment]
@cih9088 Can we close this? When can we expect to replace the NNCF version to PyPI release?
@sovrasov Do you know any info?

**Comment by cih9088** [comment]
> @cih9088 Can we close this? When can we expect to replace the NNCF version to PyPI release?

I was told that 2.4.0 would be released in a couple of weeks. Lets update to the official release at once.

**Comment by samet-akcay** [comment]
> > @cih9088 Can we close this? When can we expect to replace the NNCF version to PyPI release?
> 
> I was told that 2.4.0 would be released in a couple of weeks. Lets update to the official release at once.

@cih9088 , just an update, nncf v2.4.0 has just been released

**Comment by samet-akcay** [comment]
2 hours ago, actually :)
https://pypi.org/project/nncf/2.4.0/

**Pull request closed without merging**